lua_scripts/init_autogen_usr_bin_rules.lua

```diff
@@ -1,7 +1,8 @@
 -- Copyright (c) 2009,2012 Nokia Corporation.
+-- Copyright (c) 2021      Jolla Ltd.
 -- Author: Lauri T. Aarnio
 --
--- Licensed under MIT license
+-- License: LGPL-2.1
```

Comment:
License change from MIT to LGPL?

Reply:
The main license of the app, I figured that when we touch the code we should align the lua code that's under MIT.

Reply:
I'm not sure that's really a good idea. While technically it's not illegal (note: I'm not a lawyer, so don't blame me when someone comes and sues us), it only serves to create confusion. I mean, the lua script was licensed to us under the MIT license. The fact that we change the text in the file to say "LGPL" doesn't change the fact that it's still under MIT license. The next person who gets to read the code just does not know it, unless they read through the git history.